.circleci/config.yml

```diff
@@ -31,6 +31,21 @@ jobs:
             docker push janx/chromium
           no_output_timeout: 90m
 
+  discourse:
+    docker:
+      - image: docker:stable
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          command: |
+            cd discourse
+            docker login -u $DOCKER_USER -p $DOCKER_PASS
+            docker pull janx/ubuntu-dev
+            docker build -t janx/discourse -f discourse.dockerfile .
+            docker push janx/discourse
+          no_output_timeout: 90m
```

Comment:
Nit: This image probably doesn't need a lot of resources to build, so I'd prefer having it as a Docker Hub automated build directly (like [Janitor and Kresus images](https://hub.docker.com/u/janitortechnology/), for example) rather than having CircleCI build and deploy it (like Servo and Chromium images, which need beefier hardware).

To create a Docker Hub automated build for Discourse on Janitor, you simply need to:
- put the project dockerfile in a GitHub repository somewhere (having it in this repo is fine, or you could also create a dedicated repo if you'd like)
- then create an "automated build" on Docker Hub that points to the dockerfile in the repo (e.g. under https://hub.docker.com/u/janitortechnology/) which will auto-build whenever the repo is changed
- then also link this automated build to the `janx/ubuntu-dev` image directly on Docker Hub, so that the Discourse image is rebuilt also when the `janx/ubuntu-dev` base image is changed.

I can help with any of these steps, e.g. by giving you access to the "janitortechnology" org on Docker Hub.